pom.xml

```diff
@@ -184,6 +184,7 @@
     <module>mongodb</module>
     <module>testing</module>
     <module>junit</module>
+    <module>junit-extension</module>
```

Comment:
You could move this into the [java8-features](https://github.com/togglz/togglz/blob/a0d1817cfd5adf746115b3e508abe914654b90f2/pom.xml#L300-L308) profile. This is the way we are currently handling modules which require JDK8.